pkg/apprepo/apprepos_handler.go

```diff
@@ -65,7 +70,7 @@ type appRepositoriesHandler struct {
 	kubeappsNamespace string
 
 	// The Kubernetes client using the pod serviceaccount
-	svcKubeClient *kubernetes.Clientset
+	svcKubeClient coreClientsetInterface
```

Comment:
rather than defining our own interface, isn't it better to use `kubernetes.Interface` instead?

Reply:
Yep, you're right. I'm used to defining smaller interfaces with just the functionality we need to be able to test, but in this case the test client already exists, so no need.